### PR TITLE
Merge 7.0 to 8.0

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:  Thank you for the forward port!

**Comment by sbidoul** [comment]
Conflict resolution looks ok. :+1:

File: .coveragerc [hunk 1]
```diff
@@ -3,6 +3,7 @@ include =
     */bank-statement-reconcile/*
```
Comment (pedrobaeza) [inline .coveragerc]: This file is not needed anymore.

**Comment by pedrobaeza** [comment]
A minor thing. The rest LGTM.

File: .coveragerc (hunk 1 above)
Comment (lepistone) [inline .coveragerc]: Well spotted @pedrobaeza, I removed the file.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**